src/Mobs/PathFinder.cpp

```diff
@@ -0,0 +1,233 @@
+
+
+#include "Globals.h"
+#include "PathFinder.h"
+#include "../Chunk.h"
+
+cPathFinder::cPathFinder(double a_MobWidth, double a_MobHeight)
+	: m_Path(nullptr)
+	, m_GiveUpCounter(0)
+	, m_TicksSinceLastPathReset(1000)
```

Comment:
You mean recalculating solely based on distance as you proposed earlier in `MoveToPosition`?